.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
   - EXAMPLES_TEST=true
 
 script:
-  - export BAZEL_OPTS=--show_progress_rate_limit=5.0
```

Comment:
I think this may be better to keep as an env variable rather than modifying files (which is also not idempotent), but up to you.

Reply:
I think it's worth keeping for the gain of simplifying the shell scripts, given that the file modification is only on Travis (and `script` only happens once).